index.html

```diff
@@ -776,11 +781,15 @@ <h4>Script Type</h4>
                 the document represents the result of the adaptation of an <a>Original Language Transcript</a> or
                 a <a>Translated Transcript</a> for recording, e.g. for better lip-sync in a dubbing workflow,
                 or to ensure that the words can fit within the time available in an audio description workflow.</p>
-                <p><a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Text</a> objects
-                  in the <a>Target Recording Language</a>.
-                  They MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
-                  in the case that their language is not the <a>Target Recording Language</a>, 
-                  for context, to assist further processing.</p>
+                <p><a>Script Events</a> in this type of <a>script</a>:</p>
+                <ul>
+                  <li>SHOULD contain <a>Text</a> objects
+                    in the <a>Target Recording Language</a>;</li>
+                  <li>MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
+                    in the case that their language is not the <a>Target Recording Language</a>, 
+                    for context, to assist further processing;</li>
```

Comment:
I think we should harmonize that text with the one for as-recorded script that uses a different wording. 
```
                  <li>MAY also contain <a>Original</a> <a>Text</a> objects from
                    the <a>Original Language Transcript</a>
                    or <a>Translation</a> <a>Text</a> objects in other languages 
                    for context and quality verification;</li>
```

Reply:
Related issue to this comment: #244.